### DISCUSSION TITLE
Is there a way to expand the width of the dialog?

### DISCUSSION THREAD

**Discussion by hragbalian** [reflex-dev/reflex#3110] [Q&A] [answered]
I've tried modifying the size of the content to "4" and inserting an inline style modifying "width", but neither seem to have any effect.

**Comment by picklelo** [comment, marked as answer]
@hragbalian It looks like they're setting a `max_width` prop on the content - I was able to make it wider like this:

```python
rx.dialog.root(
    rx.dialog.trigger(rx.button("Open Dialog")),
    rx.dialog.content(
        rx.dialog.title("Welcome to Reflex!"),
        rx.dialog.description(
            "This is a dialog component. You can render anything you want in here.",
        ),
        rx.dialog.close(
            rx.button("Close Dialog", size="3"),
        ),
        max_width="100vw", # Make this a bigger value, it's fixed by default.
        width="100vw",
    ),
)
```

**Reply by hragbalian** [reply]
Thanks @picklelo! That works, though I actually had to lower the value to 50vw to get the size I wanted, which was wider than the default.  Strange, but it works!
